common/changes/@binaris/shift-db-client/split-db_2019-08-18-08-47.json

```diff
@@ -3,7 +3,7 @@
     {
       "packageName": "@binaris/shift-db-client",
       "comment": "New client code for accessing over Concord",
-      "type": "patch"
+      "type": "minor"
```

Comment:
This is the wrong `changes` file since we no longer have a `shift-db-client` package, but only shift-db. I think `common/changes/@binaris/shift-db-client` can be removed since it will have no corresponding package.